### ISSUE TITLE
PublicClientApplication.OperatingSystemAccount equivalent for msal4j-brokers

### ISSUE THREAD

**Issue by flowchart87** [AzureAD/microsoft-authentication-library-for-java#678]
Hi,

according to https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token-wam

There should be the possibibility to sign-in with the user currently logged in Windows.

in MSAL.NET this can be easily achieved through this:
```c#
PublicClientApplication.OperatingSystemAccount
```  

Is there anyhow the possibility to do the same through MSAL4J-brokers ?

**Comment by Avery-Dunn** [comment]
Hello @flowchart87 : You can check this page for more information on using that package: https://github.com/AzureAD/microsoft-authentication-library-for-java/wiki/Using-Web-Account-Manager-with-MSAL-Java

For normal MSAL Java, the `acquireTokenSilently` call will just try to retrieve a token from the cache. When the broker is enabled using the `MSALRuntimeBroker` from that msal4j-brokers package, `acquireTokenSilently` behaves a bit differently:
- It will try to retrieve a token from a WAM-related cache instead of MSAL Java's normal cache
- If there's is no cached token, it will try to perform a silent login+token call using the default operating system user

In short, when using a broker `acquireTokenSilently` encompasses all silent auth options, including using the OS account
